edgelet/support-bundle/src/support_bundle.rs

```diff
@@ -143,8 +143,11 @@ where
         future::ok(self)
             .and_then(Self::write_check)
             .and_then(Self::write_module_logs)
-            .and_then(Self::write_edgelet_log)
-            .and_then(Self::write_docker_log)
+            .and_then(|s| s.write_system_log("aziot-keyd", "aziot-keyd"))
+            .and_then(|s| s.write_system_log("aziot-certd", "aziot-certd"))
+            .and_then(|s| s.write_system_log("aziot-identityd", "aziot-identityd"))
+            .and_then(|s| s.write_system_log("iotedged", "iotedge"))
+            .and_then(|s| s.write_system_log("docker", "docker"))
```

Comment:
In a separate method (something like `get_modules` where there is an `include_ms_only` flag), can we build a list of system services we want to query and use the iotedge version as a way to determine that list?

Reply:
Maybe, but it sounds like more effort than it's worth. While it is possible to run one version of iotedge CLI against a different version of iotedged, they're currently packaged together so it's not very likely. And the code in this PR is backwards-compatible.